.github/workflows/publish-to-bcr.yml

```diff
@@ -0,0 +1,38 @@
+# Publishes to the Bazel Central Registry.
+#
+# Based on .github/workflows/publish.yaml from aspect-build/rules_lint v1.3.5.
+# See .bcr/README.md.
+name: Publish to the Bazel Central Registry
+
+on:
+  # Run from release.yml.
+  workflow_call:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+    secrets:
+      bcr_publish_token:
+        required: true
+
+  # In case of problems, enable manual dispatch from the GitHub UI.
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+
+jobs:
+  publish-to-bcr:
+    uses: bazel-contrib/publish-to-bcr/.github/workflows/publish.yaml@v0.0.3
```

Comment:
I suggest using the latest `v0.1.0`.

Reply:
@kormide Thanks, done in https://github.com/bazelbuild/rules_scala/pull/1731/commits/7f2594d9baff5f27abe3d3323b9dc282f25c13f7.

BTW, I see [release_ruleset.yaml is at v7.2.2](https://github.com/bazel-contrib/.github/releases/tag/v7.2.2). Is it worth updating to that, too? At a first quick glance, it doesn't look like the API has changed.

Reply:
Probably not load bearing for your case, but it does have some fixes so worth upgrading.

Reply:
Thanks, also done in https://github.com/bazelbuild/rules_scala/pull/1731/commits/b27b6b582727c0a4b87ec04fabc970786ae75b30.